src/video/win32_v.cpp

```diff
@@ -993,6 +993,25 @@ void VideoDriver_Win32Base::EditBoxLostFocus()
 	SetCandidatePos(this->main_wnd);
 }
 
+std::vector<int> VideoDriver_Win32Base::GetListOfMonitorRefreshRates()
+{
+	std::vector<int> rates = {};
+	EnumDisplayMonitors(nullptr, nullptr, [](HMONITOR hMonitor, HDC hDC, LPRECT rc, LPARAM data) -> BOOL {
+		auto& list = *reinterpret_cast<std::vector<int>*>(data);
```

Comment:
`auto &` instead of `auto& `

Reply:
Why exactly?

Reply:
Coding style. We put `*` and `&` next to the variable, not next to the type.